### PR TITLE
feat: [#184834138] split content into summary and detailed descriptio

### PR DESCRIPTION
**Pull request by somabadri**
…n fields

## Description

Split the content field in the license tasks into two fields, the summary description and the detailed description.

### Ticket

[184834138](https://www.pivotaltracker.com/story/show/184834138)

## Code author checklist

- [x] I have performed a self-review of my code
- [x] I have created and/or updated relevant documentation, if necessary
- [x] I have not used any relative imports
- [x] I have checked for and removed instances of unused config from CMS
- [x] I have pruned any instances of unused code
- [x] I have not added any markdown to labels, titles and button text in config

### CODE REVIEW

**Review by adidner — APPROVED** [review]
Seems good to me

File: content/src/roadmaps/license-tasks/apply-for-food-truck-license.md [hunk 1]
```diff
@@ -11,13 +11,12 @@ webflowType: business-license
...
 licenseCertificationClassification: LICENSE
+summaryDescriptionMd: "Food trucks need to be cleaned and sanitized according to the guidelines from your local Department of Health. The regulation and licensing of food trucks vary by location.
+You'll need to check license requirements with the county and/or local Department of Health in every county where you plan to do business."
 ---
-Food trucks need to be cleaned and sanitized according to the guidelines from your local Department of Health. The regulation and licensing of food trucks vary by location. 
 
-You'll need to check license requirements with the county and/or local Department of Health in every county where you plan to do business.
-
-- - -
+---
```
Comment (jphechter) [inline content/src/roadmaps/license-tasks/apply-for-food-truck-license.md:18]: I can't tell if this is the diff playing tricks on my eyes or not. We only need one `---` right? Is this 2 in a row?

File: content/src/roadmaps/license-tasks/apply-for-shop-license.md [hunk 2]
```diff
@@ -16,20 +16,20 @@ callToActionText: Apply for My Cosmetology Shop License
 webflowType: business-license
 webflowId: 5f77298fd2749a78af9cf7df
 licenseCertificationClassification: LICENSE
+summaryDescriptionMd: "New shops, relocated shops, or shops that have transferred ownership are required to obtain a license. After your application is submitted and reviewed, your shop will be inspected."
 ---
-New shops, relocated shops, or shops that have transferred ownership are required to obtain a license. After your application is submitted and reviewed, your shop will be inspected.
 
-- - -
+---
```
Comment (jphechter) [inline content/src/roadmaps/license-tasks/apply-for-shop-license.md:22]: Same as above. I think you can drop this.

File: content/src/roadmaps/license-tasks/appraiser-certification.md [hunk 3]
```diff
@@ -13,10 +13,9 @@ callToActionText: "Apply ​for My Appraiser Certification "
 webflowType: individual-license
 webflowId: 6414793bb48ba7e9395c9540
 licenseCertificationClassification: CERTIFICATION
+summaryDescriptionMd: "You need a certification to appraise real estate with one to four residential units—regardless of value or complexity. If you are opening an `Appraisal Management Company|appraisal-management-company` you must have a certified appraiser."
 ---
```
Comment (jphechter) [inline content/src/roadmaps/license-tasks/appraiser-certification.md:17]: Same as above.

**Review by jphechter — COMMENTED** [review]
I'm seeing a number of instances where you're using 2 sequential lines of `---` in the markdown files. I see our "metadata" section being wrapped like that, I also see us generating `<hr>` using `- - -`. I don't know if Netlify treats these functionally different or not. Just want to validate because we now have sequential horizontal line breaks. 

I figured this also might also be a preservation of current behavior as you're rendering the summary and content sections side-by-side.

Can you confirm these are intentional?

**Comment by somabadri** [comment]
> I'm seeing a number of instances where you're using 2 sequential lines of `---` in the markdown files. I see our "metadata" section being wrapped like that, I also see us generating `<hr>` using `- - -`. I don't know if Netlify/Webflow/Navigator treats these functionally different or not. Just want to validate because we now have sequential horizontal line breaks.
> 
> I figured this also might also be a preservation of current behavior as you're rendering the summary and content sections side-by-side.
> 
> Can you confirm these are intentional?

Yes, this was intentional. The content team wanted to be able to add or delete the `<hr>` from the detailed description section, that's why there is two `---`, one to signal the end of metadata and the other which renders as a `<hr>`.

**Pull request merged**